react/utils/dateRules.js

```diff
@@ -0,0 +1,61 @@
+import moment from 'moment'
+import msk from 'msk'
+
+export function filterDateType(fields) {
+  return fields.filter(rule => rule.type === 'date')
```

Comment:
🤔Hmmm... I'm not sure about the magic value 'date'. Maybe separate into a constant for rule types, such as:
```
RULE_TYPES = {
  DATE = 'date',
  // other types, if exists
}`
Or, if only exists a type date `const DATE_RULE: 'date'` would be just fine IMO.

Reply:
You are right. I should have chaged this in the previous PR. This one is just a rebase to the 3.x branch. However, I'll keep that in mind to a refactor we are planning for this repo. Is that okay for you?

Reply:
Absolutely